unittests/CppInterOp/CMakeLists.txt

```diff
@@ -30,9 +30,24 @@ target_link_libraries(DynamicLibraryManagerTests
   PRIVATE
   clangCppInterOp
   )
+set_source_files_properties(DynamicLibraryManagerTest.cpp PROPERTIES COMPILE_DEFINITIONS
+  "LLVM_BINARY_DIR=\"${LLVM_BINARY_DIR}\""
+  )
+set_source_files_properties(DynamicLibraryManagerTest.cpp PROPERTIES COMPILE_DEFINITIONS
+  "CPPINTEROP_VERSION=\"${CPPINTEROP_VERSION}\""
+  )
```

Comment:
Do we use LLVM_BINARY_DIR or CPPINTEROP_VERSION in DynamicLibraryManagerTest.cpp?

Reply:
These symbols are necessary so that we can include CppInterOp.cpp (to compile properly) and use from it functions that are not visible through the .h file ie. should be available only for the tests.